docs/general/administration/hardware-acceleration/apple.md

```diff
@@ -0,0 +1,94 @@
+---
+uid: admin-hardware-acceleration-apple
+title: Apple Mac
+---
+
+# HWA Tutorial On Mac
+
+This tutorial guides you on setting up full video hardware acceleration on Apple Macs via VideoToolbox.
+
+## Acceleration Methods
+
+[VideoToolbox](https://developer.apple.com/documentation/videotoolbox) is the only available method on macOS.
+
+To achieve full acceleration, [Metal](https://developer.apple.com/metal/) and OpenCL is also required.
+
+## Tone-mapping Methods
+
+Hardware accelerated HDR/DV to SDR tone-mapping is supported on most Macs from 2017 and later, with the exception of the MacBook Air (13-inch, 2017).
+
+There are two different methods that can be used. Pros and cons are listed below:
+
+1. **OpenCL**
+
+   - Pros - Supports Dolby Vision P5, detailed fine-tuning options.
+
+   - Cons - Slower, requires more powerful GPUs.
+
+2. **VideoToolbox Native**
+
+   - Pros: Lower power consumption, less dependency on GPU performance, good visual quality without manual fine-tuning.
+
+   - Cons: Lack of tuning options, potential performance issues on Intel Macs.
```

Comment:
Is there any reason for using dashes for OpenCL but colons for VideoToolbox Native? They should be the same and follow the style of other documentation pages.
Cons for VideoToolbox Native should also include lack of support for DoVi P5.